publishing/required-and-sample-data/README.md

```diff
@@ -0,0 +1,17 @@
+# Required and sample data
+
+The app should not create required nor sample data without the user's consent.
+
+## Required data
+
+If the app requires any data to operate (for example a list of the world's countries in the `Country` table), it should present a nagging information with a button that creates the data:
+
+- on the top of the main page
+- possibly in every view that needs the required data
+
+## Sample data
+
+If the app offers some nice-to-have sample data, it should offer a button that creates this data:
+
+- in the dismissible welcome message on the main page
```

Comment:
I would mention that dismissible once, not per user. Usually, a dismissable message is per user.